docs/configs/pdf/conf.py

```diff
@@ -54,7 +54,7 @@
 
 # General information about the project.
 project = u'Daml SDK'
-copyright = u'Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved. Any unauthorized use, duplication or distribution is strictly prohibited.'
+copyright = u'© Copyright 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved. Any unauthorized use, duplication or distribution is strictly prohibited. "Digital Asset" and "Daml" are Registered in the U.S. Patent and Trademark Office.'
```

Comment:
Apologies for the nitpicking here, but the rest of the repo uses `Copyright (c)`. Any reason to diverge from that? (Should we update the rest of the repo to use © instead?)

Reply:
I just went with what was there. I really don't mind what it is as long as legal is satisfied and it doesn't look glitchy. I've changed it to (c).